webpack/ForemanWebhooks/Routes/Webhooks/Components/WebhookForm/Components/ForemanFormikField.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field as FormikField } from 'formik';
+import { Spinner } from '@patternfly/react-core';
+import { TypeAheadSelect } from 'patternfly-react';
```

Comment:
I don't use Foreman's `TypeAheadSelect` because it doesn't support re-initialization and it was much simplier to use the common one.